main.scss

```diff
@@ -1,12 +1,14 @@
 @import "o-normalise/main";
-
+@import "o-visual-effects/main";
 @import "src/scss/variables";
 @import "src/scss/tooltip";
 
 
 @if ($o-tooltip-is-silent == false) {
 	 .o-tooltip {
 		@include oTooltip;
+		transition: opacity $o-tooltip-animation-duration $o-visual-effects-transition-fade,
+								margin $o-tooltip-animation-duration $o-visual-effects-transition-slide;
```

Comment:
It might be better to move the position using a translate instead of `margin` as margin can cause a layout/paint in the browser, whereas translations only effect the compositing making the animation a lot faster (Paul Lewis has a good article on this: https://aerotwist.com/blog/pixels-are-expensive/#avoiding-performance-bottlenecks)